pkg/registry/controller/rest.go

```diff
@@ -47,14 +48,25 @@ func (rcStrategy) NamespaceScoped() bool {
 func (rcStrategy) PrepareForCreate(obj runtime.Object) {
 	controller := obj.(*api.ReplicationController)
 	controller.Status = api.ReplicationControllerStatus{}
+
+	// We could allow users to initialize the sequenceNumbers, but since
+	// we don't allow them to update it, doing so would be non-intuitive.
+	controller.Spec.SequenceNumber = 0
```

Comment:
The initial controller.Spec.SequenceNumber should be set to 1, since the controller hasn't yet ack'ed the initial state.